spec/services/provider_interface/reference_status_report_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::ReferenceStatusReport do
+  let(:reference_const) { 'Reference' }
+  let(:provider) { instance_double(Provider) }
```

Comment:
We can replace these with our [factories](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/spec/factories/provider.rb) – same with the reference ones below!